packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -559,7 +593,7 @@ class ReactSixteenAdapter extends EnzymeAdapter {
     if (!inst || !this.isValidElement(inst)) {
       return false;
     }
-    return typeof inst.type === 'function' || isForwardRef(inst);
+    return typeof inst.type === 'function' || isForwardRef(inst) || isSuspense(inst);
```

Comment:
I think `isCustomComponentElement` should be renamed after adding `Suspense`, but cannot figure out a better name :(

Reply:
this is basically `isNonHostElement`, but let's not worry about the name now